src/styles/utils/ModalStyleUtils.ts

```diff
@@ -0,0 +1,273 @@
+                    overflow: 'hidden',
+                    ...getCenteredModalStyles(styles, windowWidth, isSmallScreenWidth, true),
+                };
+                swipeDirection = undefined;
+                animationIn = isSmallScreenWidth ? 'slideInRight' : 'fadeIn';
+                animationOut = isSmallScreenWidth ? 'slideOutRight' : 'fadeOut';
+                shouldAddTopSafeAreaMargin = !isSmallScreenWidth;
+                shouldAddBottomSafeAreaMargin = !isSmallScreenWidth;
+                shouldAddTopSafeAreaPadding = isSmallScreenWidth;
+                shouldAddBottomSafeAreaPadding = false;
+                break;
+            case CONST.MODAL.MODAL_TYPE.CENTERED_SMALL:
+                // A centered modal that takes up the minimum possible screen space on all devices
+                modalStyle = {
+                    ...modalStyle,
+                    ...{
+                        alignItems: 'center',
+                    },
+                };
+                modalContainerStyle = {
+                    boxShadow: '0px 0px 5px 5px rgba(0, 0, 0, 0.1)',
+                    borderRadius: 12,
+                    borderWidth: 0,
+                };
+
+                // Allow this modal to be dismissed with a swipe down or swipe right
+                swipeDirection = ['down', 'right'];
+                animationIn = 'fadeIn';
+                animationOut = 'fadeOut';
+                shouldAddTopSafeAreaMargin = false;
+                shouldAddBottomSafeAreaMargin = false;
+                shouldAddTopSafeAreaPadding = false;
+                shouldAddBottomSafeAreaPadding = false;
+                break;
+            case CONST.MODAL.MODAL_TYPE.BOTTOM_DOCKED:
+                modalStyle = {
+                    ...modalStyle,
+                    ...{
+                        alignItems: 'center',
+                        justifyContent: 'flex-end',
+                    },
+                };
+                modalContainerStyle = {
+                    width: '100%',
+                    borderTopLeftRadius: 20,
+                    borderTopRightRadius: 20,
+                    paddingTop: 12,
+                    justifyContent: 'center',
+                    overflow: 'hidden',
+                };
+
+                shouldAddBottomSafeAreaPadding = true;
+                swipeDirection = undefined;
+                animationIn = 'slideInUp';
+                animationOut = 'slideOutDown';
+                break;
+            case CONST.MODAL.MODAL_TYPE.POPOVER:
+                modalStyle = {
+                    ...modalStyle,
+                    ...popoverAnchorPosition,
+                    ...{
+                        position: 'absolute',
+                        alignItems: 'center',
+                        justifyContent: 'flex-end',
+                    },
+                };
+                modalContainerStyle = {
+                    borderRadius: 12,
```

Comment:
Not using the correct radius value of `componentBorderRadiusLarge` i.e. 16 instead of 12 caused #42716